horace_core/algorithms/@cut_data_from_file_job/private/put_sqw_data_npix_and_pix_to_file_.m

```diff
@@ -41,7 +41,7 @@
     if exist(outfile,'file') == 2
         delete(outfile);
     end
-    fid=fopen(outfile,'A');    % no automatic flushing: can be faster
+    fid=fopen(outfile,'A','l','Windows-1252');    % no automatic flushing: can be faster
```

Comment:
As mentioned before, I agree with @cmarooney-stfc . If we want to read N bytes, we should use the appropriate length types to read them rather than setting an encoding. Any of the following (https://uk.mathworks.com/help/matlab/ref/fread.html#btp1twt-1-precision):
```
uint8
int8
integer*1
uchar
unsigned char
ubit8
schar
signed char
bit8
char*1
```
Would work. It should match the type you're actually trying to read to make it clear, I would recommend `integer*1` since this is the appropriate MATLAB type.

Reply:
it is fully portable. The suggestion applies to read, and I applied the formatting to open, making `char == char*1`. There are subtle differences in parsing read data for different types of input and I do not want suddently hit these subtletly